Beats/winlogbeat/_meta/fields.yml

```diff
@@ -657,6 +657,11 @@ winlog.process.thread.id:
   name: winlog.process.thread.id
   type: long
 
+winlog.process.hash:
+  name: winlog.process.hash
+  type: object
+  description: ""
+
```

Comment:
This field is not defined in the [Winlogbeat taxomony](https://www.elastic.co/guide/en/beats/winlogbeat/current/exported-fields-winlog.html)

I suggest removing it:
```suggestion
```